Softeq.XToolkit.Common/Collections/ObservableKeyGroupsCollection.cs

```diff
@@ -21,7 +23,8 @@ namespace Softeq.XToolkit.Common.Collections
     public sealed class ObservableKeyGroupsCollection<TKey, TValue>
```

Comment:
Optional: there is a fix for ObservableRangeCollection which only exists in PH1 branch - perhaps it should be copied to the master branch. The fix is in the commit 4460e006dc4fd3510230f0ae6860adb168cb064a